runtests.sh

```diff
@@ -82,6 +83,7 @@ function print_usage {
     echo "    --coverage        : report testing code coverage, to be used with \"--net\", \"--unittests\""
     echo "    -q, --quick       : skip long running unit tests and integration tests"
     echo "    --net             : perform integration testing"
+    echo "    --disttests       : perform distributed unit testing"
```

Comment:
Should move this to line 82 for consistency with the sequence in which options are listed (in line 57).